Bindings/Java/Matlab/tests/CMakeLists.txt

```diff
@@ -34,21 +34,21 @@ macro(OpenSimAddMatlabTest TESTNAME MFILE)
                 TEST_ARGS CONFIGURATIONS ${cfg}
                 ADDITIONAL_PATH "${CMAKE_CURRENT_SOURCE_DIR}/../Utilities"
                 NO_UNITTEST_FRAMEWORK
+                UNITTEST_PRECOMMAND "cd('${matlab_output_dir}')"
                 )
-            #UNITTEST_PRECOMMAND "cd('${matlab_output_dir}')"
-            set_tests_properties(Matlab_${TESTNAME}_${cfg} PROPERTIES
-                ENVIRONMENT "OPENSIM_USE_VISUALIZER=0")
-            if(WIN32)
-                # Must set PATH so that osimJavaJNI can find osimCommon, etc.
-                set_property(TEST Matlab_${TESTNAME}_${cfg} APPEND PROPERTY
-                    ENVIRONMENT "PATH=${CMAKE_BINARY_DIR}/${cfg}")
+        if(WIN32)
+            # Must set PATH so that osimJavaJNI can find osimCommon, etc.
+            set_tests_properties(Matlab_${TESTNAME}_${cfg} PROPERTIES ENVIRONMENT
+                    "PATH=${CMAKE_BINARY_DIR}/${cfg}")
```

Comment:
It looks like the stuff above should have one more indentation.

Also, it looks like some stuff was deleted that shouldn't have been deleted (setting the OPENSIM_USE_VISUALIZER environment variable).